apps/jetbrains.py

```diff
@@ -479,6 +480,7 @@ def build_text_action_keymap():
             idea("action CodeCompletion", "action CodeCompletion")
         ],  # perfect == extra complete
         "smart": [idea("action SmartTypeCompletion")],
+        "docs": idea("action QuickJavaDoc"),
```

Comment:
I have this and other pop-ups as "pop ..." around line 733

Reply:
cool thanks.